sql_generators/experiment_monitoring/templates/templating.yaml

```diff
@@ -1,18 +1,52 @@
 queries:
   experiments_daily_active_clients_v1:
     per_app: false
+    destination_dataset: telemetry_derived
   experiment_enrollment_aggregates_live_v1:
     per_app: false
+    destination_dataset: telemetry_derived
   experiment_search_aggregates_v1:
     per_app: false
+    destination_dataset: telemetry_derived
   experiment_enrollment_aggregates_v1:
     per_app: false
+    destination_dataset: telemetry_derived
   experiment_events_live_v1:
     per_app: true
     start_date: "2023-10-10"
   experiment_search_events_live_v1:
     per_app: true
     start_date: "2023-10-10"
+  experiment_crash_events_live_v1:
+    per_app: true
+    start_date: "2025-01-01"
+    skip_applications:
+      - monitor_cirrus
```

Comment:
I guess it's easy enough to update these skip lists with new Cirrus applications when we add them to `applications` later in this file, but if we're skipping these because we just won't have data for these Cirrus apps then would it make more sense to have a wildcard filter somewhere to exclude all `*_cirrus` applications?